controller/tests/array_action/xiv/array_mediator_xiv_tests.py

```diff
@@ -91,6 +91,61 @@ def test_create_volume__generate_volume_response_raise_exception(self, response)
         with self.assertRaises(Exception):
             self.mediator.create_volume("vol", 10, [], "pool1")
 
+    def test_copy_volume_from_snapshot_succeeds_with_resize(self):
```

Comment:
please align all tests function to be similar to the name of the function which they test

Reply:
I don't see any change here.
if you think no change is needed, [please comment in the conversation before resolving](https://github.com/IBM/ibm-block-csi-driver/pull/158#discussion_r432949827), otherwise it would be harder to keep track.
I think this test function name should be changed for the same reasons that the tested function name was changed, and for naming consistency.

Reply:
it is only [reflected now](https://github.com/IBM/ibm-block-csi-driver/pull/158/commits/dd709c9821bba60e66d76b8679f7e6b367d25f38).
so just avoid resolving before ensuring it is reflected in the "files" tab diff